array_filter.json

```diff
@@ -35,11 +35,16 @@
                     },
                     {
                         "name": "index",
-                        "description": "The zero-based index of the current element being processed.",
+                        "description": "The zero-based index or the label of the current element being processed. Labels take preference over indices.",
```

Comment:
See above comment against `array_apply` - we should not let the backend decide whether to pass label or index - this should be unambiguously and explicitly defined and clear to the user. I suggest adding a `label` parameter to the callback, which will be populated with the label in case the input array is labeled.

Reply:
See above.